spacy/tests/test_cli.py

```diff
@@ -1207,3 +1207,65 @@ def test_walk_directory():
         assert (len(walk_directory(d, suffix="iob"))) == 2
         assert (len(walk_directory(d, suffix="conll"))) == 3
         assert (len(walk_directory(d, suffix="pdf"))) == 0
+
```

Comment:
Please reformat this!

Reply:
Done! Something's been off with my "format on save" I have to debug.